modules/internal/DefaultSparse.chpl

```diff
@@ -269,6 +293,28 @@ module DefaultSparse {
       for e in data[1..dom.nnz] do yield e;
     }
 
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
+      const numElems = dom.nnz;
+      const numChunks = _computeNumChunks(numElems): numElems.type;
+      if debugDefaultSparse {
+        writeln("DefaultSparseArr standalone: ", numChunks, " chunks, ",
+                numElems, " elems");
+      }
+      if numChunks <= 1 {
+        for i in 1..numElems {
+          yield data[i];
+        }
+      } else {
+        coforall chunk in 1..numChunks {
+          const (startIx, endIx) =
+            _computeChunkStartEnd(numElems, numChunks, chunk);
+          for e in data[startIx..endIx] do
+            yield e;
+        }
+      }
+    }
+
+
```

Comment:
Really minor style issue: This is pretty inconsistent with when `{}` vs `do` are used.

e.g. the debug if has braces in the arr iter, but not the dom, second for in arr iter uses `do` but first has braces